staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -98,13 +102,26 @@ type EventList struct {
 // categories are logged.
 type Policy struct {
 	metav1.TypeMeta
+	// ObjectMeta is included for interoperability with API infrastructure.
+	// +optional
+	metav1.ObjectMeta
```

Comment:
Ok, all the arguments I had with my previous pull https://github.com/kubernetes/kubernetes/pull/45765. I don't argue against this, quite the opposite!